### PR TITLE
PCHR-2621: Implement user menu in ssp

### PR DESCRIPTION
**Pull request by AkA84**
In this PR the `CRM_HRCore_UserMenuMarkup` class is implemented in the SSP theme. The result is visually the same, but the code underneath got cleaned to remove anything related to the old user menu

Related PRs:
* https://github.com/civicrm/civihr/pull/2236
* https://github.com/compucorp/civihr-employee-portal/pull/370

### CODE REVIEW

File: civihr_default_theme/scss/base/_variables.scss [hunk 1]
```diff
@@ -173,3 +173,16 @@ $crm-customselect-padding-right: $crm-customselect-caret-width + 12 !default;
...
 
 $crm-ui-select-top-padding: 7px;
+
+
+//== User Menu
+//
+//##
+$chr-user-menu-color: $light-pale;
+$chr-user-menu-line-height: inherit;
+$chr-user-menu-minify-breakpoint: 1250px;
```
Comment (reneolivo) [inline civihr_default_theme/scss/base/_variables.scss]: why not use the value for `$crm-menu-responsive-breakpoint` defined in this same variable file?

File: civihr_default_theme/scss/civihr_default_theme.style.scss [hunk 2]
```diff
@@ -106,15 +107,19 @@
...
+// --------------------------------------------------
+@import "SCSSROOT/org.civicrm.shoreditch/scss/bootstrap/overrides/style/button-groups";
 @import "SCSSROOT/org.civicrm.shoreditch/scss/bootstrap/components/custom-select";
 @import "SCSSROOT/org.civicrm.shoreditch/scss/bootstrap/components/date-picker";
 @import "SCSSROOT/org.civicrm.shoreditch/scss/bootstrap/components/text-angular";
 @import "SCSSROOT/org.civicrm.shoreditch/scss/bootstrap/components/wysiwyg";
 @import "SCSSROOT/org.civicrm.shoreditch/scss/bootstrap/overrides/style/ui-select";
 @import "SCSSROOT/org.civicrm.shoreditch/scss/bootstrap/variants/modals-xs-screen";
 
+@import "SCSSROOT/org.civicrm.bootstrapcivihr/scss/components/outside-namespace/user-menu";
```
Comment (reneolivo) [inline civihr_default_theme/scss/civihr_default_theme.style.scss:121]: why the extra space around this particular import and none of the others?

**Review by reneolivo — CHANGES_REQUESTED** [review]

File: civihr_default_theme/scss/civihr_default_theme.style.scss (hunk 2 above)
Comment (AkA84) [reply civihr_default_theme/scss/civihr_default_theme.style.scss:121]: dunno, just a visual way to separate the extensions. You think it's pointless?

Comment (reneolivo) [reply civihr_default_theme/scss/civihr_default_theme.style.scss:121]: No, it is good actually. I didn't notice the different extensions at first, but now that you mention it it's clear.

**Review by reneolivo — APPROVED** [review]

**Pull request merged**